cmd/prototype.go

```diff
@@ -381,58 +391,62 @@ func bindPrototypeFlags(cmd *cobra.Command, proto *prototype.SpecificationSchema
 	}
 }
 
-func expandPrototype(proto *prototype.SpecificationSchema, flags *pflag.FlagSet, templateType prototype.TemplateType, componentName string) (string, error) {
+func expandPrototype(proto *prototype.SpecificationSchema, templateType prototype.TemplateType, params map[string]string, componentName string) (string, error) {
+	template, err := proto.Template.Body(templateType)
+	if err != nil {
+		return "", err
+	}
+	template = append([]string{`local params = std.extVar("__ksonnet/params").components.` + componentName + ";"}, template...)
+
+	tm, err := jsonnet.Parse(componentName, strings.Join(template, "\n"))
+	text, err := tm.Evaluate(params)
```

Comment:
If we're not putting LSP-style snippets params in (_e.g._, `${foo}`) there's no need to `Evaluate`. See my comment about just returning a string in snippet.go.